File: 백준/Bronze/4153. 직각삼각형/직각삼각형.java
import java.util.*;
import java.io.*;
public class Main {
    public static void main(String[] args) throws IOException{
        BufferedReader br=new BufferedReader(new InputStreamReader(System.in));
        while(true){
            StringTokenizer st=new StringTokenizer(br.readLine());
            int a=Integer.parseInt(st.nextToken());
            int b=Integer.parseInt(st.nextToken());
            int c=Integer.parseInt(st.nextToken());
            if(a==0 && b==0 && c==0) break;
            ArrayList<Integer> list=new ArrayList<>();
            list.add(a);
            list.add(b);
            list.add(c);
            Collections.sort(list);
            if(list.get(2)*list.get(2)==list.get(0)*list.get(0)+list.get(1)*list.get(1)){ System.out.println("right"); }
            else { System.out.println("wrong"); }
        }
    }
}

File: 백준/Bronze/4153. 직각삼각형/직각삼각형.py
result=[]
while True:
    a,b,c=map(int,input().split())
    length=[a,b,c]
    length.sort()
    if a==0 and b==0 and c==0:
        break;
    else:
        if length[2]**2==length[1]**2+length[0]**2:
            result.append("right")
        else:
            result.append("wrong") 
for i in result:
    print(i)